### PR TITLE
ansible-scylla-monitoring: Allow adding a custom dockerd configuration

### PR DESCRIPTION
**Pull request by igorribeiroduarte**

### CODE REVIEW

File: ansible-scylla-monitoring/tasks/docker.yml [hunk 1]
```diff
@@ -39,6 +39,10 @@
       ./kill-all.sh
       sudo ./kill-all.sh
 
+  - name: test
+    shell: |
+      sleep 10000 
+
```
Comment (vladzcloudius) [inline ansible-scylla-monitoring/tasks/docker.yml:45]: Could you, please, explain what this patch and change a bit more?

**Review by vladzcloudius — CHANGES_REQUESTED** [review]
Rogue patch?

**Review by vladzcloudius — CHANGES_REQUESTED** [review]
Patch description: I thought "Fixes" is the marker that should be used in this case and not "Closes".

File: ansible-scylla-monitoring/defaults/main.yml [hunk 2]
```diff
@@ -89,3 +89,7 @@ prometheus_url: 'https://github.com/prometheus/prometheus/releases/download/v2.3
 
 # Additional arguments to pass to the prometheus binary
 #prometheus_args: "--web.enable-admin-api --storage.tsdb.retention.size=50G"
+
+# Use this variable for setting any custom configuration allowed by dockerd (https://docs.docker.com/reference/cli/dockerd/#on-linux)
+# The content of this variable will be coppied to /etc/docker/daemon.json
+docker_daemon_custom_config: { "bip": "172.17.0.1/24" }
```
Comment (vladzcloudius) [inline ansible-scylla-monitoring/defaults/main.yml:95]: Should be commented out, no?

**Review by vladzcloudius — CHANGES_REQUESTED** [review]

File: ansible-scylla-monitoring/tasks/docker.yml [hunk 3]
```diff
@@ -2,6 +2,12 @@
 - name: Populate service facts
   service_facts:
 
+- name: Add custom dockerd config
+  copy:
+    content: "{{ docker_daemon_custom_config }}"
+    dest: "/etc/docker/daemon.json"
+  become: true
```
Comment (vladzcloudius) [inline ansible-scylla-monitoring/tasks/docker.yml:9]: conditioning is missing: `when docker_daemon_custom_config is defined ...`

**Review by vladzcloudius — CHANGES_REQUESTED** [review]

File: ansible-scylla-monitoring/tasks/docker.yml (hunk 3 above)
Comment (igorribeiroduarte) [reply ansible-scylla-monitoring/tasks/docker.yml:9]: Fixed

File: ansible-scylla-monitoring/defaults/main.yml (hunk 2 above)
Comment (igorribeiroduarte) [reply ansible-scylla-monitoring/defaults/main.yml]: Fixed

**Review by vladzcloudius — APPROVED** [review]

**Pull request merged**